### PR TITLE
Vector Tiles

### PR DESCRIPTION
**Pull request by bagnell**
This has already been reviewed in #4665. This PR is to merge to master.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Signed CLA is on file.

@bagnell, thanks for the pull request! Maintainers, we have a signed CLA from @bagnell, so you can review this at any time.

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

**Comment by mramato** [comment]
@bagnell whose account are these sample datasets under?  All beta.cesium.com assets in Sandcastle should use the same account so they can be centrally managed.

**Comment by bagnell** [comment]
They are under my account. I didn't know which the others belonged to. Can you transfer them?

**Comment by pjcozzi** [comment]
I know not all of this is code, but does this set the record for largest Cesium PR?

image

File: CHANGES.md [hunk 1]
```diff
@@ -10,6 +10,21 @@ Change Log
...
 * Added `Cartesian3.projectVector` for projecting one vector to another [#6093](https://github.com/AnalyticalGraphicsInc/cesium/pull/6093) 
 * Added `Cesium3DTileset.tileFailed` event that will be raised when a tile fails to load. The object passed to the event listener will have a url and message property. If there are no event listeners, error messages will be logged to the console. [#6088](https://github.com/AnalyticalGraphicsInc/cesium/pull/6088)
+* Added `AttributeCompression.zigZagDeltaDecode` which will decode delta and ZigZag encoded buffers in place.
+* Added `pack` and `unpack` functions to `OrientedBoundingBox` for packing to and unpacking from a flat buffer.
+* Added `Cesium3DTileset.ellipsoid` determining the size and shape of the globe. This can be set at construction and defaults to a WGS84 ellipsoid.
+* Added experimental support for [3D Tiles Vector Data](https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/vector-tiles/TileFormats/VectorData) ([#4665](https://github.com/AnalyticalGraphicsInc/cesium/pull/4665)). The new and modified Cesium APIs are:
```
Comment (pjcozzi) [inline CHANGES.md]: And geometry tiles.  Please update in master.

**Comment by bagnell** [comment]
> I know not all of this is code, but does this set the record for largest Cesium PR?

Nope, but it's up there. The 3D Tiles PR:
3d_tiles_pr

**Comment by pjcozzi** [comment]
Ah, tough competition. 👍

File: CHANGES.md (hunk 1 above)
Comment (bagnell) [reply CHANGES.md]: Updated. 662a833

**Pull request merged**